resources/teamcity/android/keyman-android-release.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+# Keyman is copyright (C) SIL Global. MIT License.
+#
+# TC build script to build release of Keyman for Android.
+
+# shellcheck disable=SC2164
+# shellcheck disable=SC1091
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../resources/build/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+# shellcheck disable=SC2154
+. "${KEYMAN_ROOT}/resources/shellHelperFunctions.sh"
+. "${KEYMAN_ROOT}/resources/teamcity/includes/tc-download-info.inc.sh"
+. "${KEYMAN_ROOT}/resources/teamcity/includes/tc-helpers.inc.sh"
+. "${KEYMAN_ROOT}/resources/teamcity/android/android-actions.inc.sh"
+
+################################ Main script ################################
+
+builder_describe \
+  "Build release of Keyman for Android" \
+  "all            run all actions" \
+  "build          configure and build Keyman for Android" \
+  "publish        publish release" \
+  "--fv                               additionally build FV Android" \
```

Comment:
Conceptually, we have the idea of OEM builds. Only FV at present but wanted to keep it open for more. We could go with `--oem=OEM` or similar, where `$OEM` could be a comma separated list e.g. `firstvoices`. But that's possibly unnecessary.

Reply:
I added a `ENHANCE` comment.